### PR TITLE
netperf initial commit

### PR DESCRIPTION
**Pull request by juanman2**
An MRM for deploying docker onto Apcera platform.

@Kusanagi9999 @cpk55

### CODE REVIEW

**Comment by cpk55** [comment]
this is a good one to test network performance. But should this go into sample-apps? Also need to reword jenkins with netperf in the readme.

**Comment by juanman2** [comment]
Fixed README copy pasta.  Yes, I think why not add examples of how to use Docker?

File: netperf/README.MD [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# About
+
+Contains the scripts and multi-resource manifest to deploy netperf & netserver in an Apcera cluster.
+
+
+# Using Multi-Resource Manifest
+With the multi-resource manifest you can establish a single descriptive declarative file that can deploy the netserver and its netperf client, including the network, and in a single command deploys the application end-to-end. Please see the file deploy.sh for details, also look at the cleanup.sh script that cleans up the work that the deploy script did.
```
Comment (Kusanagi9999) [inline netperf/README.MD]: With the multi-resource manifest you can establish a declarative file that can deploy the netserver and its netperf client (end-to-end and including the network) in a single command.

File: netperf/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# This scripts cleans up the objects creatd by the deploy of jenkins-manifest.json
```
Comment (Kusanagi9999) [inline netperf/cleanup.sh]: typo: created

File: netperf/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+if [ -z "$1" ]; then
+   echo "Please specify the cluster name."
+   print_usage
+else
+   export CLUSTERNAME=$1
+fi
+
+if [ -z "$2" ]; then
+   echo "Please specify the namespace"
```
Comment (Kusanagi9999) [inline netperf/deploy.sh]: nit: period like above?

File: netperf/netperf.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
+  "jobs": {
+    "job::${NAMESPACE}::netserver": {
+      "docker": {
+        "image": "paultiplady/netserver:latest"
```
Comment (Kusanagi9999) [inline netperf/netperf.json]: latest has the risk that things might break in the future without us noticing but maybe that's OK here?

**Comment by juanman2** [comment]
Implemented @Kusanagi9999 feedback.  Merging.

**Pull request merged**